src/main/java/dk/frankbille/scoreboard/domain/User.java

```diff
@@ -26,6 +26,7 @@
 	private String username;
 	private Player player;
 	private League defaultLeague;
+	private RatingCalculatorType ratingCalculator;
```

Comment:
Ahh I see. Maybe the player page is wrong already? It doesn't really make sense that there is a rating on the player page. It is across all leagues, so doesn't really make sense.